CHANGELOG.md

```diff
@@ -58,6 +58,13 @@ file an issue, so that we can see how to handle this.
 * https://github.com/R3BRootGroup/R3BRoot/pull/413
 
 
+## 18.8.1 (UNRELEASED) - 2023-01-XX
+
+### Breaking Changes
+* Check the return value of `source->InitUnpackers()`/`source->ReinitUnpackers()`
+  in `FairRunOnline`. Stop run if `false` returned.
```

Comment:
Would you really call this a breaking change? The return value wasn't used before and is now used. It could even be considered a bugfix?

In any case it will only break one use case: People return `false` and want things to continue. But why did they return `false` in the first place?